profiler/shakesapp/go.mod

```diff
@@ -0,0 +1,12 @@
+module shakesapp
```

Comment:
Would you mind making this the full path?

`github.com/GoogleCloudPlatform/golang-samples/profiler/shakesapp`

Otherwise, it can run into trouble using other packages from this repo, being imported, etc. Neither of those are important for this sample, but it's good to do in general (this has bitten us in the past).

Reply:
Thanks, done.